src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
```

Comment:
Let's specify that it's a price for 1 day.
Something like `dayPrice`/`priceForDay` would be okay.